pzsd_bot/cogs/points/admin.py

```diff
@@ -0,0 +1,307 @@
+import logging
+from enum import Enum, auto
+from itertools import batched
+
+from discord import ApplicationContext, Bot, Embed, default_permissions
+from discord.commands import option
+from discord.ext.commands import Cog, slash_command
+from discord.ext.pages import Paginator
+from sqlalchemy import insert, select, update
+
+from pzsd_bot.db import Session
+from pzsd_bot.model import pzsd_user
+from pzsd_bot.settings import PointsSettings
+
+logger = logging.getLogger(__name__)
+
+
+class NameState(Enum):
+    VALID_NAME = auto()
+    INVALID_NAME = auto()
+    DISALLOWED_NAME = auto()
+
+
+class PointUserAdmin(Cog):
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    def validate_name(self, name: str) -> NameState:
```

Comment:
static method

Reply:
yeaaaa I guess, but i've started to wonder what's even the point. All it does is mean i can omit `self` from the parameter list, that's literally it.